flang/test/Semantics/OpenACC/acc-declare-validity.f90

```diff
@@ -14,7 +14,7 @@ module openacc_declare_validity
 
   !$acc declare create(aa, bb)
 
-  !ERROR: 'aa' in the CREATE clause is already present in another clause in this module
+  !WARNING: 'aa' in the CREATE clause is already present in the same clause in this module
```

Comment:
Can you also add an error check for case when it is in conflicting declare clause again?

Reply:
There was on in the function but I added one back in the module declaration.